tensorvision/utils.py

```diff
@@ -54,12 +58,14 @@ def set_dirs(hypes, hypes_fname):
         else:
             runs_dir = os.path.join(base_path, 'RUNS')
 
-        if not FLAGS.save:
+        if not FLAGS.save and FLAGS.name is None:
```

Comment:
FLAGS.save is `false` on default [unless $TV_SAVE=true]. My though was, that if someone explicitly uses --name , that he indeed wants to give the run a name. Even if he does not use --save. Do you think we should change this?